src/Core/Application/RealTime/UseCase/FindHost/FindHostPresenterInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * Copyright 2005 - 2021 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Core\Application\RealTime\UseCase\FindHost;
+
+use Core\Application\Common\UseCase\PresenterInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+interface FindHostPresenterInterface extends PresenterInterface
+{
+    /**
+     * @param FindHostResponse $response
+     * @return void
+     */
+    public function present(FindHostResponse $response): void;
+
+    /**
+     * @return Response
+     */
+    public function show(): Response;
```

Comment:
update controller too
```suggestion
    public function show(): \Object;
```